docs/src/sysimages.md

```diff
@@ -119,6 +125,10 @@ REPL and package manager not require compilation after a Julia restart.. It is
 therefore unlikely that `incremental=false` is of much use unless in special
 cases for sysimage creation (for apps it is a different story though).
 
+As an alternative consider using another mechanism to pass the `-J` flag to
+Julia as above. These include creating a desktop shortcut or a shell alias,
+`$ alias julia='julia -q -J/path/to/sysimage.so'`, that includes the option.
+
```

Comment:
@KristofferC I recommend creating an alias in the last paragraph of the section.